### PR TITLE
Update getting-service-phone-numbers.md

### PR DESCRIPTION
**Pull request by edfishermsft**
Corrected typo in title, "Service" was spelled "Srvice" and removed internal comment from public facing push.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/getting-service-phone-numbers.md [hunk 1]
```diff
@@ -24,9 +24,8 @@ ms.custom:
 description: Learn how to get new phone numbers and port or transfer existing numbers for Audio Conferencing, Auto Attendants, and Call Queues (service numbers) for Teams.
 ---
 
-# Srvice phone numbers for Calling Plans
+# Service phone numbers for Calling Plans
 
-**NOTE TO REVIEWERS: THIS SEEMS SPECIFIC TO CALLING PLANS. SHOULD I MOVE IT UNDER THE CALLING PLANS NODE?**
```
Comment (JohanFreelancer9) [inline Teams/getting-service-phone-numbers.md:29]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @edfishermsft, kindly commit my suggested change to remove the extra blank line. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
These fixes were just submitted in another PR.  Closing this PR.

**Pull request closed without merging**